### DISCUSSION TITLE
Position of separate line

### DISCUSSION THREAD

**Discussion by NEO33333333** [SubtitleEdit/subtitleedit#7709] [General]
How to choose the position of each separate line of dialogue in the same line of subtitle?

**Comment by niksedk** [comment]
Where and what are you doing? 
What subtitle format?

See https://www.nikse.dk/subtitleedit/help#good-github-issue

**Comment by NEO33333333** [comment]
Format Ebu .stl

"-How are you?
                                      -I'm fine."

Just Like this.

It's for SDH Subtiles

Le mar. 5 déc. 2023 à 21:45, Nikolaj Olsson ***@***.***> a
écrit :

**Comment by niksedk** [comment]
Sorry, SE does not support this.

**Comment by NEO33333333** [comment]
Okay, thanks for your response🙏🙏

Le mer. 6 déc. 2023 à 09:57, Nikolaj Olsson ***@***.***> a
écrit :
